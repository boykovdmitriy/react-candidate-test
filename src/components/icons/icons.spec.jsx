import React from 'react';
import {shallow} from 'enzyme';
import {
  BackIcon,
  ClockIcon,
  FavoriteIcon,
  HomeIcon,
  ListIcon,
  RepeatIcon,
  SearchIcon,
  TvIcon,
  UserIcon
} from './index';

describe('Icons', () => {
  it('render BackIcon', () => {
    const tree = shallow(<BackIcon/>);
    expect(tree).toMatchSnapshot();
  });

  it('render ClockIcon', () => {
    const tree = shallow(<ClockIcon/>);
    expect(tree).toMatchSnapshot();
  });

  it('render FavoriteIcon', () => {
    const tree = shallow(<FavoriteIcon/>);
    expect(tree).toMatchSnapshot();
  });

  it('render HomeIcon', () => {
    const tree = shallow(<HomeIcon/>);
    expect(tree).toMatchSnapshot();
  });

  it('render ListIcon', () => {
    const tree = shallow(<ListIcon/>);
    expect(tree).toMatchSnapshot();
  });

  it('render RepeatIcon', () => {
    const tree = shallow(<RepeatIcon/>);
    expect(tree).toMatchSnapshot();
  });

  it('render SearchIcon', () => {
    const tree = shallow(<SearchIcon/>);
    expect(tree).toMatchSnapshot();
  });

  it('render TvIcon', () => {
    const tree = shallow(<TvIcon/>);
    expect(tree).toMatchSnapshot();
  });

  it('render UserIcon', () => {
    const tree = shallow(<UserIcon/>);
    expect(tree).toMatchSnapshot();
  });
});