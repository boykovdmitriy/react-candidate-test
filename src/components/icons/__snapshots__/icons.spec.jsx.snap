// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons render BackIcon 1`] = `
<i
  className="fa fa-arrow-left"
/>
`;

exports[`Icons render ClockIcon 1`] = `
<i
  className="far fa-clock"
/>
`;

exports[`Icons render FavoriteIcon 1`] = `
<i
  className="fa fa-star"
/>
`;

exports[`Icons render HomeIcon 1`] = `
<i
  className="fa fa-home"
/>
`;

exports[`Icons render ListIcon 1`] = `
<i
  className="fa fa-th-list"
/>
`;

exports[`Icons render RepeatIcon 1`] = `
<i
  className="fa fa-repeat "
/>
`;

exports[`Icons render SearchIcon 1`] = `
<i
  className="fa fa-search"
/>
`;

exports[`Icons render TvIcon 1`] = `
<i
  className="fa fa-tv"
/>
`;

exports[`Icons render UserIcon 1`] = `
<i
  className="fa fa-user"
/>
`;
